src/styles/index.scss

```
@@ -1,3 +1,113 @@
 body {
   margin: 0;
 }
+
+p {
+  margin: 0;
+}
+
+h3 {
+  margin: 0;
+}
+
+.cards {
+  --color: #00acdc;
+  --color-second: #fff;
+
+  box-sizing: border-box;
+  color: #060b35;
+  width: 200px;
+  padding: 32px 16px 16px;
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  border: 1px solid #f3f3f3;
+
+  &__foto {
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin: 0 auto 40px;
+  }
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+  &__code {
+    margin-top: 4px;
+    margin-bottom: 16px;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+
+    color: #616070;
+  }
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+  &__price {
+    color: #616070;
+    margin-top: 24px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    line-height: 18px;
+    &-text {
+      font-size: 12px;
+      font-weight: 400;
+    }
+    &-price {
+      color: #060b35;
```

Add the blank line between the selectors
```suggestion
    line-height: 18px;
    
    &-text {
      font-size: 12px;
      font-weight: 400;
    }
    
    &-price {
      color: #060b35;
```